.eslintrc.js

```diff
@@ -102,7 +102,6 @@ module.exports = {
     'no-catch-shadow': 'off',
     'no-cond-assign': ['error', 'always'],
     'no-confusing-arrow': 'error',
-    'no-console': 'error',
```

Comment:
don't remove `no-console`. 
Add in line 1 (`src/index.ts`) the following comment
```
/* eslint-disable no-console */
```